extern/noah-owp-modular/CMakeLists.txt

```diff
@@ -58,18 +58,18 @@ add_compile_definitions(BMI_ACTIVE NGEN_FORCING_ACTIVE NGEN_OUTPUT_ACTIVE NGEN_A
 # "The path to the C shared object library file, libnetcdff.so or the library name if it can be found by the linker, i.e. netcdff")
 ##### END NETCDF #####
 
-set(MODEL_SOURCE_DIR noah-mp-modular/src/)
-set(BMI_SOURCE_DIR noah-mp-modular/bmi/)
+set(MODEL_SOURCE_DIR noah-owp-modular/src/)
+set(BMI_SOURCE_DIR noah-owp-modular/bmi/)
 
 file(GLOB BMI_SOURCE ${BMI_SOURCE_DIR}*.f90)
 file( GLOB MODEL_SOURCES ${MODEL_SOURCE_DIR}*.f90)
 
-list(APPEND MODEL_SOURCES noah-mp-modular/driver/AsciiReadModule.f90
-                          noah-mp-modular/driver/OutputModule.f90)
+list(APPEND MODEL_SOURCES noah-owp-modular/driver/AsciiReadModule.f90
+                          noah-owp-modular/driver/OutputModule.f90)
 
 #add_library(surfacebmi_obj STATIC ${MODEL_SOURCES} )
 #target_compile_options(surfacebmi_obj PUBLIC -cpp -ffree-line-length-none)
-#target_include_directories(surfacebmi_obj PUBLIC noah-mp-modular/modules/surface_bmi/driver)
+#target_include_directories(surfacebmi_obj PUBLIC noah-mp-modular/modules/surface_bmi/driver) #PATH IS OUT OF DATE
```

Comment:
Should probably just remove these commented out lines...

Reply:
Agreed. I wasn't sure if they served another purpose, so I'm happy to delete them if they're not needed.

Reply:
Commented lines have been removed